src/shared/components/Studio/StudioEditorForm.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Input, Form, Tooltip, Icon, Button } from 'antd';
+import { WrappedFormUtils } from 'antd/lib/form/Form';
+
+const StudioEditorForm: React.FC<{
+  form: WrappedFormUtils;
+  saveStudio?(label: string): void;
+}> = ({ form, saveStudio }) => {
+  const { getFieldDecorator } = form;
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    form.validateFields((err, values) => {
+      if (!err) {
+        saveStudio && saveStudio(values.label);
+      }
+    });
+  };
+
+  const formItemLayout = {
```

Comment:
Can we move this to a style class/less file?

Reply:
This is an antD layout implementation I don't think it can be ported to a style file. Maybe the best thing is to just remove it?

Reply:
we pass it as a js object as prop to antd form